test/models/item_test.rb

```diff
@@ -13,11 +13,46 @@ def setup
     assert item.valid?
   end
 
+  test 'it cannot have an empty string as a name' do
+    item.title = ""
+    assert item.invalid?
+  end
+
+  test "it cannot have an empty string as a description" do
+    item.description = ""
+    assert item.invalid?
+  end
+
+  test "it cannot have a duplicate title" do
+    @item1 = Item.create(title: 'espresso', description: "this is black gold", price: 30000)
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.